### PR TITLE
Proposed spelling corrections

### PR DESCRIPTION
**Pull request by jeis2497052**
are these updates OK ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,38 +54,37 @@ stages:
 unit2.7:
   image: python:2.7
   before_script:
-    - pip install -r requirements-dev.txt
-    - pip install -r requirements.txt
+	- pip install -r requirements-dev.txt
```
Comment (mterzo) [inline README.md]: These look over indented now.

Comment (jeis2497052) [reply README.md]: sorry, looks like my bad...  I will fix tomorrow

Comment (mterzo) [reply README.md]: Thanks.

Comment (jeis2497052) [reply README.md]: I think I restored the spacing Mike (sorry)

**Pull request merged**